deps/slfritchie_eflame-a08518142126f5fc541a3a3c4a04c27f24448bae/src/eflame.app.src

```diff
@@ -0,0 +1,13 @@
+{application, eflame,
```

Comment:
As my comment says, the commit# gives this information anyway.
Now, if you want this to work with releases the directory name has to match the name in app.src; or you can add a couple lines in rel/relx.config.src